tests/test_it.py

```diff
@@ -216,6 +216,23 @@ def test_I201_most_specific_imports():
     ]
 
 
+def test_I201_relative_imports():
```

Comment:
I'd split this into two tests, one for `.` and one for `..` - that way one can fail and one can pass and they're in separate tests so you get more info